src/ui/Table/Table.stories.mdx.broken

```diff
@@ -1,6 +1,6 @@
 {/* Table.mdx */}
```

Comment:
Can probably delete this file, just kept for now so we can see it tried to convert it but couldn't

Reply:
not opposed to deleting